aadii0408.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "Aaditya Champaneri || Aadi ",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Artist By Choice, Engineer By Default | Python Developer",
+  {
+  "tags": [ "Python", "Machine Learning", "Open Source", "Marketing", "Communities",]
```

Whoopsie. Trailing comma here.
```suggestion
  "tags": [ "Python", "Machine Learning", "Open Source", "Marketing", "Communities"]
```

---

Please, also move your `json` file to the `data` folder.